lib/brains/tests/test_client.py

```diff
@@ -22,7 +22,9 @@ def test_normal(self):
 
     @raises(NotFoundError)
     def test_not_found(self):
+        set_mocks(['GET', 'customers/does-not-exist', 404, ''])
```

Comment:
this is awesome. It makes it way more clear how a test gets its data. You could even roll them up in shared methods like ``setup_non_existant_customer()`` and I think it will still be clear.